lib/ain-ocean/src/model/transaction_vout.rs

```diff
@@ -1,10 +1,9 @@
 use bitcoin::Txid;
 use serde::{Deserialize, Serialize};
 
-pub type TransactionVoutKey = (Txid, usize);
-
 #[derive(Debug, Serialize, Deserialize)]
 pub struct TransactionVout {
+    pub id: String, // format!({}{:08x}, txid, usize")
```

Comment:
Why are we reverting back to String here ?

Reply:
the old one usize wasn't padded
references: https://github.com/BirthdayResearch/jellyfishsdk/blob/767aa7199458153486350a6804fa0851932d91ae/apps/whale-api/src/module.model/_hex.encoder.ts#L37

Reply:
We should fully implement `Column` trait for `TransactionVout` column instead

Reply:
Can check `TransactionByBlockHash` implementation for ref

Reply:
fixed at [feac743](https://github.com/DeFiCh/ain/pull/2834/commits/feac74377bf6dc59556672799e6c9d744ec132ed)